packages/chain-adapters/src/ChainAdapterCLI.ts

```diff
@@ -188,21 +188,29 @@ const main = async () => {
 
     // send cosmos example
     try {
+      const value = '100'
+
+      const feeData = await cosmosChainAdapter.getFeeData({ sendMax: false })
+      const fee = feeData.slow.txFee
+      const gas = feeData.slow.chainSpecific.gasLimit
+
       const cosmosUnsignedTx = await cosmosChainAdapter.buildSendTransaction({
-        to: `0x47CB53752e5dc0A972440dA127DCA9FBA6C2Ab6F`,
-        value: '1',
+        to: 'cosmos1j26n3mjpwx4f7zz65tzq3mygcr74wp7kcwcner',
+        value,
         wallet,
         bip44Params: cosmosBip44Params,
-        chainSpecific: { gas: '0' }
+        chainSpecific: { gas, fee }
       })
-      const cosmosSignedTx = await cosmosChainAdapter.signTransaction({
+
+      if (!cosmosChainAdapter.signAndBroadcastTransaction) return
```

Comment:
is there ever a case where this function isn't defined? extra check doesn't hurt anything but wondering if I'm missing something

Reply:
This function is only used for wallets that cannot sign offline (ie metamask), so technically there could be a time where it isn't defined.